### DISCUSSION TITLE
安装pptpd找不到kmod-gre和kmod-mppe依赖

### DISCUSSION THREAD

**Discussion by zzxysun** [coolsnowwolf/lede#9712] [R68S / 电犀牛专区]
安装pptpd找不到kmod-gre和kmod-mppe依赖，能不能在下个版本里增加pptp vpn。或者提供kmod-gre和kmod-mppe依赖安装包，让我们自己安装。

**Comment by ouhaiorg** [comment]
m68s OpenWrt R22.6.1 碰到同样的问题
